shell/platform/darwin/ios/platform_view_ios.h

```diff
@@ -55,12 +56,15 @@ class PlatformViewIOS : public PlatformView {
                      const std::string& main,
                      const std::string& packages) override;
 
+  NSObject<FlutterBinaryMessenger>* GetBinaryMessenger();
```

Comment:
I would expect this accessor to not have `Get` in its name. It can use [underscore variable-style naming](https://google.github.io/styleguide/cppguide.html#Function_Names) because it is an accessor. It can be inlined because the implementation is trivial, e.g.

`NSObject<FlutterBinaryMessenger>* binary_messenger() { return binary_messenger_; }`

Compare to [`platform_message_router()`](https://github.com/goderbauer/engine/blob/033a3a10f5193a9ecc16904bedfcdc0bcf8ce816/shell/platform/darwin/ios/platform_view_ios.h#L38) and and [`framebuffer()`](https://github.com/goderbauer/engine/blob/033a3a10f5193a9ecc16904bedfcdc0bcf8ce816/shell/platform/darwin/ios/ios_gl_context.h#L29). 

Or if you prefer, `BinaryMessenger()` isn't *inconsistent* with the Google style guide, but I think the variable-style naming is clearer. See [`ColorSpace()`](https://github.com/goderbauer/engine/blob/033a3a10f5193a9ecc16904bedfcdc0bcf8ce816/shell/platform/darwin/ios/ios_gl_context.h#L37).

It should also `const`, though we don't seem to be doing that consistently in this part of the code base.

Reply:
Changed to your suggestion. This fits in better with the other code.